RC-0.18.1 (#1916)
microsoft/vscode-cosmosdb: CHANGELOG.md

# Change Log

> ## 0.18.2 - 2021-07-30

Reviewer: This should be `0.18.1` - we just remove the `-alpha` part from the version for a patch release

### Fixed

> - [Fix for Timing Out issue when loading Postgres Flexible Servers](https://github.com/microsoft/vscode-cosmosdb/issues/1913)

Reviewer: Instead of making this whole sentence a link, I'd recommend doing something like [this](https://github.com/microsoft/vscode-azurestorage/blob/main/CHANGELOG.md#fixed-7) and link just the issue at the end:

```suggestion
- Fix time out issue when loading Postgres Flexible Servers [#1913](https://github.com/microsoft/vscode-cosmosdb/issues/1913)
```

Reviewer 2: https://github.com/microsoft/vscode-azurefunctions/blob/main/CHANGELOG.md#0191---2019-10-15

For reference, this is how Eric usually does it

> - Fixed the ETIMEDOUT issue "...when signing the PostgreSQL Flexible server on another virtual machine"

Reviewer: I don't think you have to quote the issue title exactly - something like this would be good:

```suggestion
- Fixed the ETIMEDOUT issue when signing into a PostgreSQL Flexible server on multiple machines
```
